### DISCUSSION TITLE
DNN-based approaches to nerve tractography, reconstruction and centerline extraction

### DISCUSSION THREAD

**Discussion by michelodu** [dipy/dipy#3034] [Tractography / Tractometry]
Dear members of the DiPy community, 
I am writing an NIH proposal, and I would like to pick your brains about the best options available in open source, whether DiPy-based or not, for nerve tractography. Specifically, for the intracranial portion of cranial nerves. Are there deep neural network-based approaches for tractography available publicly? Moreover, is there a way to obtain a centerline from a cluster of nerve tracts? Would that be in QuickBundles, or is there something else that you recommend? Finally, if we can compute shape statistics for these centerline, reliably, would it be feasible to integrate shape (and topology) priors into more stable nerve tractography and reconstruction?
Thanks for your kind support. Best wishes,
Michel Audette

**Comment by aldiak** [comment]
Hi, how are you? where you able to find an answer to your question?
